enterprise/backend/src/metabase_enterprise/metabot_v3/context.clj

```diff
@@ -41,19 +44,25 @@
    :keyword
    :any])
 
+(def ^:private ^DateTimeFormatter current-user-time-format
+  (DateTimeFormatter/ofPattern "'Today is' EEEE, 'Year' yyyy, 'Date' yyyy-MM-dd, HH:mm:ss"))
+
+(defn- set-user-time
+  [context]
+  (let [offset-time (or (some-> context :current_time_with_timezone OffsetDateTime/parse)
+                        (OffsetDateTime/now))]
+    (-> context
+        (dissoc :current_time_with_timezone)
+        (assoc :current_user_time (.format current-user-time-format offset-time)))))
+
 (mu/defn create-context
   "Create a tool context."
-  [_context]
-  (merge {}
-         #_(metabot-v3.tools.query/create-context context)))
-
-(mu/defn describe-context
-  "Transforms the tool context into LLM context."
-  [_context]
-  (merge {}
-         #_(metabot-v3.tools.query/describe-context context)))
+  [context]
+  (-> context
+      set-user-time
+      (select-keys [:current_user_time])))
 
 (mu/defn create-reactions
   "Extracts reactions based on the current context."
```

Comment:
Not relevant to your change, but this is unfamiliar to me, I guess at some point we decided reactions should occur by changing the context? Are you using that functionality? It feels bizarre to have two different ways of emitting a reaction from a tool (changing the context in a particular way will auto-emit the reaction, or you can do `:reactions` in your result as well)...

Reply:
I'm not using this and I'm not aware of its origins either. I agree that two mechanisms for the same thing just lead to confusing and bugs.